### DISCUSSION TITLE
When I choose a file it removes the other

### DISCUSSION THREAD

**Discussion by snibo** [btimofeev/UniPatcher#78] [Q&A] [answered]
I just downloaded the app and when I select the patch from my files it gets selected correctly and then when I select the rom the rom gets selected and the patch gets removed. No matter what I try it can only select one file at once please guys help me because I don't really know what to do because I have read all the qna and found nothing. I can't even apply my patchs online because for some reason I can't upload files anymore. I am using a Samsung J2 core on Android 8 if it helps

**Comment by btimofeev** [comment, marked as answer]
Hi.
Turn off “Don’t keep activities” under “Developer Options”

**Reply by snibo** [reply]
Oh gosh that worked and I think that it also solved a bunch of other problems.
Not. Related at all but do you know why it is totally bugged and I am stuck on the title screen
